Shared.Tests/Constants.cs

```diff
@@ -9,5 +9,7 @@ public class TestConstants
         public const string DefaultVideoUpPath = DefaultRootPath + "video/upload/";
         public const string DefaultImageUpPath = DefaultRootPath + "image/upload/";
         public const string DefaultImageFetchPath = DefaultRootPath + "image/fetch/";
+        public const string TestPublicId = "test_id";
```

Comment:
Please do not use constant public IDs and  tags, see `GetMethodTag()` and `GetUniquePublicId()`.

Constant public ids and tags will lead to misleading results when resources are not deleted(assertion failed, etc) and tests are run once again.